src/main/scala/viper/gobra/frontend/PackageResolver.scala

```diff
@@ -230,22 +230,51 @@ object PackageResolver {
     } yield pkgName
   }
 
-  private lazy val pkgClauseRegex = """(?:\/\/.*|\/\*(?:.|\n)*\*\/|package(?:\s|\n)+([a-zA-Z_][a-zA-Z0-9_]*))""".r
+  private lazy val pkgClauseRegex = """(?:\/\/.*|\/\*(?:.|\n)*?\*\/|package(?:\s|\n)+([a-zA-Z_][a-zA-Z0-9_]*))""".r
```

Comment:
please add a comment that multiline comments are lazily matches, i.e. stop at the earliest `*/`